_includes/footer.html

```diff
@@ -12,35 +12,42 @@
             <div class="col-sm-10">
                 <ul class="links">
                     <li>
-                        <span class="message">Hello from Seattle and Zürich.</span>
+                        <span class="message">Hello from Toulouse.</span>
                     </li>
                     <li>
-                        <a class="github-button" href="https://github.com/Microsoft/language-server-protocol" data-icon="octicon-star" data-show-count="true" aria-label="Star Microsoft/language-server-protocol on GitHub">Star</a>
+                        <a class="github-button" href="https://github.com/lapce/plugin-server-protocol"
+                            data-icon="octicon-star" data-show-count="true"
+                            aria-label="Star Microsoft/plugin-server-protocol on GitHub">Star</a>
                     </li>
                     <li>
-                        <a class="github-button" href="https://github.com/Microsoft/language-server-protocol/subscription" aria-label="Watch Microsoft/language-server-protocol on GitHub">Watch</a>
+                        <a class="github-button" href="https://github.com/lapce/plugin-server-protocol/subscription"
+                            aria-label="Watch lapce/plugin-server-protocol on GitHub">Watch</a>
                     </li>
                 </ul>
             </div>
             <div class="col-sm-2">
                 <ul class="links">
-                    <li style="display: none;" >
-                        <a id="footer-cookie-link" style="cursor: pointer; padding-right:20px" onclick="manageConsent()"
-                            aria-label="Manage cookies">Manage cookies</a>
-                    </li>
                     <li>
                         <div class="copyright">
-                            <a id="footer-microsoft-link" class="logo" href="https://www.microsoft.com">
+                            <a id="footer-lapce-link" class="logo" href="https://lapce.dev">
                                 <picture>
-                                    <source srcset="{{site.baseurl}}/img/microsoft-logo.png" media="(prefers-color-scheme: dark)"></source>
-                                    <img src="{{site.baseurl}}/img/microsoft-logo-inverted.png" height="20" alt="Microsoft">
+                                    <source
+                                        srcset="https://raw.githubusercontent.com/lapce/lapce/master/extra/images/logo.png"
```

Comment:
I'm cross linking to the logo on Lapce's main repo. I could be using site.baseurl, but it that case I need to also upload and publish images to this repo. As of right now, because it's not being published the same way LSP does, it would result in 404 errors